inception-concept-linking/src/main/resources/META-INF/asciidoc/user-guide/annotation_concept-linking.adoc

```diff
@@ -0,0 +1,30 @@
+[[sect_annotation_concept-linking]]
+=== Concept Linking
+==== Interactive Concept Linking
+When the user wants to link a mention, he marks it and starts typing the name of the concept.
+This triggers a pass through the entity linking pipeline and retrieves a ranked list of candidates,
+which is displayed to the user in the form of a drop-down menu.
+In order to enable a user to select the correct concept, it is crucial for him to receive textual
+information about the candidates in question. This is realized by showing an overlay text box with
+descriptions retrieved from the KB.
+
+image::concept-linking2.png[align="center"]
+
+If the typed string matches the marked surface form, the query is executed with the whole
```

Comment:
That is too specific. I would write that the suggestions get updatet on new input/while typing.